crates/sovereign-sdk/rollup-interface/src/state_machine/da.rs

```diff
@@ -33,6 +33,11 @@ impl core::cmp::Ord for SequencerCommitment {
     }
 }
 
+/// Sorts the sequencer commitments by their L2 start block number
+pub fn sort_sequencer_commitments(commitments: &mut [SequencerCommitment]) {
+    commitments.sort();
+}
```

Comment:
This should go into `crates/primitives/utils`

But, since it's a 1 liner, we can just call `.sort` on the vec directly rather than passing it here.

Reply:
lol, it works ofc. somehow my rust-analyzer was broken I guess